### ISSUE TITLE
Touch layout needs redesign

### ISSUE THREAD

**Issue by abasgames** [FWGS/xash3d-fwgs#208]
The mobile game touch layout seems very old and not practical maybe redesign it to something similar to
![68a2812c-110f-46c5-8885-df0ef584158c~2](https://user-images.githubusercontent.com/45899491/90330402-21f4fe00-dfb5-11ea-9211-750c35e9cfa9.jpg)
This

**Comment by a1batross** [comment]
I agree. Our default touch layout was based on Beloko's layout, mostly because it was comfortable and we didn't had anything better.

Modern mobile games usually have some kind of standard, like on screenshot.

I don't think it will require code changes, it's already possible with our configurable and scriptable touch controls.

**Comment by a1batross** [comment]
I agree. Our default touch layout was based on Beloko's layout, mostly because it was comfortable and we didn't had anything better.

Modern mobile games usually have some kind of standard, like on screenshot.

I don't think it will require code changes, it's already possible with our configurable and scriptable touch controls.

**Comment by GOLD11DRAGON** [comment]
yes
